script.js

```diff
@@ -28,44 +27,52 @@ function initDemo() {
         initChat(getEngagement);
     }
     initAuth0();
+
+    if(!ssoKey) {
+        var code = getURLParams(window.location.search).code;
+        if(code) {
+            ssoKey = code;
+        }
+    }
 }
 
 function initAuth0() {
-    $('docuemnt').ready(function () {
+    $('docuemnt').ready(function() {
         var lock = new Auth0Lock(AUTH0_CLIENT_ID, AUTH0_DOMAIN, {
             auth: {
                 redirectUrl: location.href,
-                responseType: 'id_token',
+                responseType: 'code',
```

Comment:
If I understand it correctly, now you only support Code flow. You should support both code and implicit flow at once.